docs/stencil-docs/storefront-customization/theme-accessibility.md

```diff
@@ -0,0 +1,148 @@
+# Developing Themes for Accessibility
+
+<div class="otp" id="no-index">
+
+### On this page
+- [Using alt text](#using-alt-text)
+- [Text accessibility](#text-accessibility)
+- [Keyboard accessibility](#keyboard-accessibility)
+- [Customization](#customization)
+- [Resources](#resources)
+</div>
+
+Designing for accessibility means to be inclusive to the needs of all users. In this tutorial, you will learn how to implement accessibility in your theme and improve theme usability for all users.
+
+We have included four accessibility best practices and examples that will help you develop a theme that meets the needs of the broadest range of abilities.
```

Comment:
remove - [We have included four accessibility best practices and examples that will help you develop a theme that meets the needs of the broadest range of abilities.]